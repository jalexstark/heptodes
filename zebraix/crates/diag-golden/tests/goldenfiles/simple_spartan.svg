<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400pt" height="300pt" viewBox="0 0 400 300">
<defs>
<g>
<g id="glyph-0-0">
<path d="M 6.4375 0 L 5.53125 0 L 5.53125 -3.328125 L 1.875 -3.328125 L 1.875 0 L 0.96875 0 L 0.96875 -7.140625 L 1.875 -7.140625 L 1.875 -4.125 L 5.53125 -4.125 L 5.53125 -7.140625 L 6.4375 -7.140625 Z M 6.4375 0 "/>
</g>
<g id="glyph-0-1">
<path d="M 2.921875 -5.453125 C 3.378906 -5.453125 3.769531 -5.351562 4.09375 -5.15625 C 4.425781 -4.957031 4.679688 -4.675781 4.859375 -4.3125 C 5.035156 -3.945312 5.125 -3.523438 5.125 -3.046875 L 5.125 -2.515625 L 1.453125 -2.515625 C 1.472656 -1.898438 1.628906 -1.4375 1.921875 -1.125 C 2.222656 -0.8125 2.640625 -0.65625 3.171875 -0.65625 C 3.515625 -0.65625 3.816406 -0.6875 4.078125 -0.75 C 4.335938 -0.8125 4.609375 -0.898438 4.890625 -1.015625 L 4.890625 -0.25 C 4.617188 -0.125 4.347656 -0.0351562 4.078125 0.015625 C 3.816406 0.0664062 3.5 0.09375 3.125 0.09375 C 2.625 0.09375 2.175781 -0.00390625 1.78125 -0.203125 C 1.394531 -0.410156 1.09375 -0.71875 0.875 -1.125 C 0.65625 -1.539062 0.546875 -2.046875 0.546875 -2.640625 C 0.546875 -3.222656 0.644531 -3.722656 0.84375 -4.140625 C 1.039062 -4.566406 1.316406 -4.890625 1.671875 -5.109375 C 2.023438 -5.335938 2.441406 -5.453125 2.921875 -5.453125 Z M 2.90625 -4.734375 C 2.488281 -4.734375 2.15625 -4.597656 1.90625 -4.328125 C 1.664062 -4.066406 1.523438 -3.691406 1.484375 -3.203125 L 4.203125 -3.203125 C 4.203125 -3.660156 4.097656 -4.03125 3.890625 -4.3125 C 3.679688 -4.59375 3.351562 -4.734375 2.90625 -4.734375 Z M 2.90625 -4.734375 "/>
</g>
<g id="glyph-0-2">
<path d="M 1.734375 0 L 0.84375 0 L 0.84375 -7.59375 L 1.734375 -7.59375 Z M 1.734375 0 "/>
</g>
<g id="glyph-0-3">
<path d="M 5.515625 -2.6875 C 5.515625 -1.800781 5.285156 -1.113281 4.828125 -0.625 C 4.378906 -0.144531 3.773438 0.09375 3.015625 0.09375 C 2.535156 0.09375 2.109375 -0.0078125 1.734375 -0.21875 C 1.367188 -0.4375 1.078125 -0.753906 0.859375 -1.171875 C 0.648438 -1.585938 0.546875 -2.09375 0.546875 -2.6875 C 0.546875 -3.570312 0.769531 -4.253906 1.21875 -4.734375 C 1.664062 -5.210938 2.273438 -5.453125 3.046875 -5.453125 C 3.523438 -5.453125 3.945312 -5.34375 4.3125 -5.125 C 4.6875 -4.914062 4.976562 -4.601562 5.1875 -4.1875 C 5.40625 -3.78125 5.515625 -3.28125 5.515625 -2.6875 Z M 1.453125 -2.6875 C 1.453125 -2.050781 1.578125 -1.546875 1.828125 -1.171875 C 2.078125 -0.804688 2.476562 -0.625 3.03125 -0.625 C 3.570312 -0.625 3.96875 -0.804688 4.21875 -1.171875 C 4.46875 -1.546875 4.59375 -2.050781 4.59375 -2.6875 C 4.59375 -3.320312 4.46875 -3.816406 4.21875 -4.171875 C 3.96875 -4.535156 3.566406 -4.71875 3.015625 -4.71875 C 2.472656 -4.71875 2.078125 -4.535156 1.828125 -4.171875 C 1.578125 -3.816406 1.453125 -3.320312 1.453125 -2.6875 Z M 1.453125 -2.6875 "/>
</g>
<g id="glyph-0-4">
</g>
<g id="glyph-0-5">
<path d="M 4.3125 -3.03125 C 4.226562 -3.300781 4.148438 -3.566406 4.078125 -3.828125 C 4.015625 -4.097656 3.96875 -4.304688 3.9375 -4.453125 L 3.90625 -4.453125 C 3.875 -4.304688 3.828125 -4.097656 3.765625 -3.828125 C 3.710938 -3.566406 3.640625 -3.296875 3.546875 -3.015625 L 2.578125 -0.015625 L 1.578125 -0.015625 L 0.109375 -5.375 L 1.015625 -5.375 L 1.765625 -2.515625 C 1.835938 -2.222656 1.90625 -1.929688 1.96875 -1.640625 C 2.03125 -1.347656 2.078125 -1.101562 2.109375 -0.90625 L 2.15625 -0.90625 C 2.175781 -1.019531 2.203125 -1.160156 2.234375 -1.328125 C 2.273438 -1.492188 2.316406 -1.664062 2.359375 -1.84375 C 2.410156 -2.03125 2.460938 -2.195312 2.515625 -2.34375 L 3.453125 -5.375 L 4.421875 -5.375 L 5.34375 -2.34375 C 5.414062 -2.125 5.484375 -1.878906 5.546875 -1.609375 C 5.617188 -1.335938 5.671875 -1.109375 5.703125 -0.921875 L 5.734375 -0.921875 C 5.753906 -1.085938 5.800781 -1.316406 5.875 -1.609375 C 5.945312 -1.898438 6.019531 -2.203125 6.09375 -2.515625 L 6.84375 -5.375 L 7.75 -5.375 L 6.265625 -0.015625 L 5.234375 -0.015625 Z M 4.3125 -3.03125 "/>
</g>
<g id="glyph-0-6">
<path d="M 3.34375 -5.453125 C 3.445312 -5.453125 3.554688 -5.445312 3.671875 -5.4375 C 3.785156 -5.4375 3.890625 -5.425781 3.984375 -5.40625 L 3.875 -4.59375 C 3.78125 -4.613281 3.679688 -4.628906 3.578125 -4.640625 C 3.484375 -4.648438 3.390625 -4.65625 3.296875 -4.65625 C 3.015625 -4.65625 2.753906 -4.578125 2.515625 -4.421875 C 2.273438 -4.273438 2.082031 -4.066406 1.9375 -3.796875 C 1.800781 -3.535156 1.734375 -3.222656 1.734375 -2.859375 L 1.734375 0 L 0.84375 0 L 0.84375 -5.359375 L 1.5625 -5.359375 L 1.671875 -4.375 L 1.703125 -4.375 C 1.878906 -4.664062 2.101562 -4.914062 2.375 -5.125 C 2.65625 -5.34375 2.976562 -5.453125 3.34375 -5.453125 Z M 3.34375 -5.453125 "/>
</g>
<g id="glyph-0-7">
<path d="M 2.75 0.09375 C 2.082031 0.09375 1.546875 -0.132812 1.140625 -0.59375 C 0.742188 -1.0625 0.546875 -1.753906 0.546875 -2.671875 C 0.546875 -3.585938 0.75 -4.28125 1.15625 -4.75 C 1.5625 -5.21875 2.097656 -5.453125 2.765625 -5.453125 C 3.171875 -5.453125 3.503906 -5.375 3.765625 -5.21875 C 4.035156 -5.070312 4.253906 -4.890625 4.421875 -4.671875 L 4.484375 -4.671875 C 4.472656 -4.753906 4.457031 -4.878906 4.4375 -5.046875 C 4.425781 -5.222656 4.421875 -5.359375 4.421875 -5.453125 L 4.421875 -7.59375 L 5.296875 -7.59375 L 5.296875 0 L 4.59375 0 L 4.453125 -0.71875 L 4.421875 -0.71875 C 4.265625 -0.488281 4.050781 -0.296875 3.78125 -0.140625 C 3.507812 0.015625 3.164062 0.09375 2.75 0.09375 Z M 2.890625 -0.625 C 3.453125 -0.625 3.847656 -0.78125 4.078125 -1.09375 C 4.316406 -1.40625 4.4375 -1.875 4.4375 -2.5 L 4.4375 -2.65625 C 4.4375 -3.320312 4.320312 -3.832031 4.09375 -4.1875 C 3.875 -4.550781 3.46875 -4.734375 2.875 -4.734375 C 2.40625 -4.734375 2.050781 -4.539062 1.8125 -4.15625 C 1.570312 -3.78125 1.453125 -3.28125 1.453125 -2.65625 C 1.453125 -2.007812 1.570312 -1.507812 1.8125 -1.15625 C 2.050781 -0.800781 2.410156 -0.625 2.890625 -0.625 Z M 2.890625 -0.625 "/>
</g>
<g id="glyph-0-8">
<path d="M 1.625 -2.015625 L 1.0625 -2.015625 L 0.8125 -7.140625 L 1.875 -7.140625 Z M 0.71875 -0.546875 C 0.71875 -0.785156 0.773438 -0.953125 0.890625 -1.046875 C 1.015625 -1.148438 1.164062 -1.203125 1.34375 -1.203125 C 1.507812 -1.203125 1.648438 -1.148438 1.765625 -1.046875 C 1.890625 -0.953125 1.953125 -0.785156 1.953125 -0.546875 C 1.953125 -0.304688 1.890625 -0.128906 1.765625 -0.015625 C 1.648438 0.0859375 1.507812 0.140625 1.34375 0.140625 C 1.164062 0.140625 1.015625 0.0859375 0.890625 -0.015625 C 0.773438 -0.128906 0.71875 -0.304688 0.71875 -0.546875 Z M 0.71875 -0.546875 "/>
</g>
</g>
</defs>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 104 281.25 C 104 276.28125 93.253906 272.25 80 272.25 C 66.746094 272.25 56 276.28125 56 281.25 C 56 286.21875 66.746094 290.25 80 290.25 C 93.253906 290.25 104 286.21875 104 281.25 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 100 262.5 L 200 240 M 100 225 L 190 262.5 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.5" stroke-miterlimit="10" d="M 190 262.5 L 20 288.75 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 142.398438 315 C 142.398438 318.199219 137.601562 318.199219 137.601562 315 C 137.601562 311.800781 142.398438 311.800781 142.398438 315 Z M 142.398438 315 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 127.761719 312.761719 L 132.238281 317.238281 M 132.238281 312.761719 L 127.761719 317.238281 Z M 132.238281 312.761719 "/>
<g fill="rgb(0%, 0%, 100%)" fill-opacity="1">
<use xlink:href="#glyph-0-0" x="160" y="288.19043"/>
<use xlink:href="#glyph-0-1" x="167" y="288.19043"/>
<use xlink:href="#glyph-0-2" x="173" y="288.19043"/>
<use xlink:href="#glyph-0-2" x="176" y="288.19043"/>
<use xlink:href="#glyph-0-3" x="179" y="288.19043"/>
<use xlink:href="#glyph-0-4" x="185" y="288.19043"/>
<use xlink:href="#glyph-0-5" x="188" y="288.19043"/>
<use xlink:href="#glyph-0-3" x="196" y="288.19043"/>
<use xlink:href="#glyph-0-6" x="202" y="288.19043"/>
<use xlink:href="#glyph-0-2" x="206" y="288.19043"/>
<use xlink:href="#glyph-0-7" x="209" y="288.19043"/>
<use xlink:href="#glyph-0-8" x="215" y="288.19043"/>
</g>
</svg>
