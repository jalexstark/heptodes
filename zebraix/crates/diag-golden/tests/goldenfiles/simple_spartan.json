{
  "sizing_scheme": "Fill",
  "axes_range": [
    -0.5,
    -0.5,
    1.5,
    3.5
  ]
}
